src/tests/ftest/util/server_utils.py

```diff
@@ -36,17 +36,54 @@
 import errno
 import yaml
 
+from command_utils import CommandWithParameters
+from command_utils import BasicParameter, FormattedParameter
 from avocado.utils import genio
 
 SESSIONS = {}
 
 DEFAULT_FILE = "src/tests/ftest/data/daos_server_baseline.yaml"
 AVOCADO_FILE = "src/tests/ftest/data/daos_avocado_test.yaml"
 
-
 class ServerFailed(Exception):
     """Server didn't start/stop properly."""
 
+class ServerCommand(CommandWithParameters):
+    """Defines a object representing a server command."""
+
+    def __init__(self, command="daos_server"):
+        """Create a server Command object"""
+
+        super(ServerCommand, self).__init__(command, shell=True, sudo=True)
+
+        self.request = BasicParameter("{}")
+        self.action = BasicParameter("{}")
+        self.targets = FormattedParameter("-t {}")
+        self.config = FormattedParameter("-o {}")
+        self.port = FormattedParameter("-p {}")
+        self.storage = FormattedParameter("-s {}")
+        self.modules = FormattedParameter("-m {}")
+        self.xshelpernr = FormattedParameter("-x {}")
+        self.firstcore = FormattedParameter("-f {}")
+        self.group = FormattedParameter("-g {}")
+        self.attach = FormattedParameter("-a {}")
+        self.sock_dir = FormattedParameter("-d {}")
+    def __str__(self):
```

Comment:
Will make change.

Reply:
It makes sense to remove overriden __str__ method from server_utils and dmg_utils since the param order is now handled in the override of the get_param_names.